src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,114 @@
+package storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import tasklist.TaskList;
+import task.*;
+
+public class Storage {
+
+    public String FILE_PATH;
+    public ArrayList<Task> botMemory;
+    public Storage(String FILE_PATH) {
```

Comment:
nit: Might be better if you kept spacing between members consistent throughout the project.